modules/core/include/opencv2/core/mat.hpp

```diff
@@ -2138,6 +2138,7 @@ template<typename _Tp> class Mat_ : public Mat
     void create(Size _size);
     //! equivalent to Mat::create(_ndims, _sizes, DatType<_Tp>::type)
     void create(int _ndims, const int* _sizes);
+	void release();
```

Comment:
Bad indentation in this line (probably tab has been used instead of spaces).
Also the new method should be commented as the previous one:
```C++
//! equivalent to Mat::release()
  void release();
```